### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kingukana**
Hugo installation for Windows.

### CODE REVIEW

**Comment by kingukana** [comment]
please review.

File: README.md [hunk 1]
```diff
@@ -16,9 +16,12 @@ To verify your install, run this command:
 ```
 hugo version
 ```
```
Comment (IamMujuziMoses) [inline README.md:18]: You can modify step 1 to look like this;
Screenshot (3)

Comment (IamMujuziMoses) [reply README.md:18]: you can use `visit this link` instead  of `go to this link`

**Comment by kingukana** [comment]
Hi,
Thank you for the feedback, i will modify it as soon as possible

**Review by heralden — APPROVED** [review]
Thanks! This will be helpful for users on other platforms.

**Pull request merged**